packages/ui/src/components/Modal/modal.tsx

```
@@ -74,7 +74,7 @@ const Modal: React.FC<ModalProps> = (props) => {
             className={classes}
             closeIcon={<Cross/>}
             footer={footer || null}
-            title={<Typography.Title level={3} ellipsis={{ rows: 2 }} children={title}/>}
+            title={<Typography.Title level={3} ellipsis={{ rows: 4 }} children={title}/>}
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)